integrations/docker/images/chip-cirque-device-base/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:focal
+
+ARG OT_BR_POSIX_CHECKOUT=master
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PLATFORM ubuntu
+ENV DOCKER 1
+ENV OT_BR_POSIX_CHECKOUT=${OT_BR_POSIX_CHECKOUT}
+
+RUN mkdir /app
+
+WORKDIR /app
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y sudo git ca-certificates psmisc dhcpcd5 wpasupplicant wireless-tools \
+  && ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
+  && git clone https://github.com/openthread/ot-br-posix . \
+  && git checkout $OT_BR_POSIX_CHECKOUT \
+  && git submodule update --init --depth=1 \
+  && ./script/bootstrap \
+  && ./script/setup \
+  && chmod 644 /etc/bind/named.conf.options \
+  && mv ./script /tmp \
+  && mv ./etc /tmp \
+  && find . -delete \
+  && rm -rf /usr/include \
+  && mv /tmp/script . \
+  && mv /tmp/etc . \
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false git ca-certificates \
```

Comment:
I'd suggest building in a throwaway container instead of purging. There's no need to come up with a list of what to remove in that case.

Reply:
I reviewed the build procedure and needs some time to figure out the otbr-posix build procedure and which package is required if we will using a throwaway container. So I will add a comment and left for follow up.